cfme/tests/openstack/cloud/test_volumes.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for Openstack cloud volumes"""
+
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils import testgen
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate([OpenStackProvider], scope='module')
+
+pytestmark = [pytest.mark.usefixtures("setup_provider_modscope")]
+
+VOLUME_SIZE = 1
+
+
+@pytest.yield_fixture(scope='function')
+def volume(appliance, provider):
+    collection = appliance.collections.volumes
+    volume = collection.create(name=fauxfactory.gen_alpha(),
+                               storage_manager='{} Cinder Manager'.format(provider.name),
```

Comment:
The storage_manager is only a field value in 58z, and is getting passed as a fill value. This raises NoSuchElement exceptions in 57z, because the form widget doesn't exist.

Reply:
Added version.pick workaround for 5.7 here